langchain/llms/openai.py

```diff
@@ -150,6 +153,10 @@ class BaseOpenAI(BaseLLM):
     """Maximum number of retries to make when generating."""
     streaming: bool = False
     """Whether to stream the results or not."""
+    allowed_special: Union[Literal["all"], AbstractSet[str]] = (set(),)
```

Comment:
don't think this should be tuple (same with disallowed_special)

Reply:
You're right. Thanks for pointing out.

Reply:
I have updated the pr